solidity/contracts/bridge/Wallets.sol

```diff
@@ -299,4 +304,36 @@ library Wallets {
 
         emit NewWalletRegistered(ecdsaWalletID, walletPubKeyHash);
     }
+
+    /// @notice Reports about a proven fraud committed by the given wallet.
+    /// @param walletPubKeyHash 20-byte public key hash of the wallet
+    /// @dev Requirements:
+    ///      - Wallet must be in Live state
+    function notifyWalletFraudProven(
+        Data storage self,
+        bytes20 walletPubKeyHash
```

Comment:
In the [RFC](https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-1.adoc#fraud-proofs) there is a parameter called `fraud_slashing_amount: The amount of stake slashed from each member of a wallet for a fraud`.
We will have to pass this parameter here, so that we know how much we need to slash each operator, right?
Perhaps this parameter should be moved from the `Frauds` library to `Wallets`?

Reply:
Yep, we can move that parameter from `Frauds` here within #133

Reply:
Please ignore this comment for now. I think I misunderstood the idea of what is the responsibility of `notifyWalletFraudProven`